functional-tests/src/test/groovy/specs/app/HomeSpec.groovy

```diff
@@ -13,22 +13,22 @@ import pages.app.modal.EditProfileModal
 
 import pages.external.ExternalLinkPage
 
+import modules.FooterModule
 import spock.lang.Unroll
 import spock.lang.Title
 import spock.lang.Stepwise
 
 @Title("Functional tests for the Home page")
 @Stepwise
-class HomeSpec extends GebReportingSpec {
+class HomeSpec extends LoggedInSpec {
+
   def setupSpec() {
-    to LoginPage
-    login("admin", System.getenv("ADMINPW"))
-    to HomePage
   }
 
   @Unroll
   def "Navigate Page from: HomePage, click Link: #ClickLink, Assert Page: #AssertPage"() {
     given: "I start on the HomePage"
+      sleep(1000)
```

Comment:
Is this to avoid errors due to timeouts? Could we use _waitFor()_ instead (or something similar) in order to make it more robust in case the required timeout is longer?

Reply:
yes ... have been talking about it with Nick -- not sure what might improve odds of success. Haven't invested a great deal of time in identifying the race condition.